src/sqlite3pp.h

```diff
@@ -165,6 +165,7 @@ namespace sqlite3pp
     int bind(int idx, char const* value, copy_semantic fcopy);
     int bind(int idx, void const* value, int n, copy_semantic fcopy);
     int bind(int idx, std::string const& value, copy_semantic fcopy);
+    int bind_blob(int idx, std::string const& value, copy_semantic fcopy);
```

Comment:
Use void* version bind for blob. Same for get.

Reply:
The thing is, it's easier to write
```
db.bind_blob(1, f());
```
rather than
```
std::string tmp = f();
db.bind(1, static_cast<void const *>(tmp.data()), tmp.size());
```

As for the getter, I don't see any downside to setting string length to the length of the string in the database rather than scanning for the null character.

Reply:
It seems a more of style choices, and I chose to have all bind in same method name, bind.